admin/routes/restart.js

```diff
@@ -5,10 +5,12 @@ const router = express.Router()
 
 const {
   getRestartOverview,
-  getSelectRestartList
+  getSelectRestartList,
+  postSubmitRestartList
 } = require('../controllers/restart')
```

Comment:
I thought we agreed we would require the whole of the controller and then go for a naming convention [controller].[method]

Reply:
For the purpose of routes I don't see it making any difference, we are not testing them after all.

Reply:
The controller however has changed.

Reply:
It's a naming convention we are implementing everywhere else, so I believe it would be beneficial also for the sake of consistency, plus you will most likely need all the controller methods for the router anyway.

Reply:
Personally I think it's extra work across all routes (it's a little though I know) for no benefit.
Don't all routes at the moment use the exact same pattern?
I would agree that destructuring makes testing harder in cases like the controllers but overall we shouldn't exclude the assignment as it improves readability in some cases.
What's the opinion of the rest on this? @GuyHarwood @jon-shipley

Reply:
I would rather it be consistent, everywhere.

Reply:
@GuyHarwood updated.
We should also create a subtask in tech debt PBI to update all the routes

Reply:
@GeorgeChatzigiannis PBI created under tech debt.